.github/workflows/release-v2.yml

```diff
@@ -557,7 +557,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        repo: [ 'stargate/jsonapi', 'riptano/c2']
+        repo: [ 'riptano/c2']
```

Comment:
Thanks for approval @tatu-at-datastax. I did just add this one additional change so that we don't dispatch event to JSON API repo for v2.0 releases (switching to using v2.1 for JSON API).